build.gradle

```diff
@@ -63,7 +67,7 @@ def distributions = [
                          "url": "https://bintray.com/openhab/mvn/download_file?file_path=org%2Fopenhab%2Fdistro%2Fopenhab-online%2F2.0.0.b4%2Fopenhab-online-2.0.0.b4.tar.gz",
                          "path": buildDir.getAbsolutePath() + '/' + 'openhab-online-2.0.0.b4.tar.gz',
                          "conflicts": "openhab2-offline",
-                         "version": "2.0.0~b5",
+                         "version": "2.0.0b5",
```

Comment:
Hm, the bintray version names don't have the tilde (https://bintray.com/openhab/apt-repo2/openhab2). But the deb version includes the tilde :-(